packages/@lwc/engine-core/src/framework/base-lightning-element.ts

```diff
@@ -190,6 +191,7 @@ type RefNodes = { [name: string]: Element };
 const refsCache: WeakMap<RefVNodes, RefNodes> = new WeakMap();
 
 export interface LightningElement extends HTMLElementTheGoodParts, AccessibleElementProperties {
+    constructor: LightningElementConstructor;
```

Comment:
Removing the `@ts-ignore` from the prototype assignment revealed that this was missing from the `LightningElement` interface.